code/modules/shuttle/shuttle.dm

```diff
@@ -297,6 +299,15 @@
 	initial_engines = count_engines()
 	current_engines = initial_engines
 
+	for (var/turf/T in all_turfs)
+		for(var/obj/effect/shuttle_sound_location/shuttle_loc in T.contents)
+			sound_source = shuttle_loc
+		if (sound_source)
+			break
```

Comment:
Why can there only be one sound source? Large shuttles would want it in multiple places

Reply:
If it plays from multiple sources/per source, it can end up being louder than 100% because byond sucks. If you're instead just going to play it from the nearest, might as well just check engines/doors like before.